packages/ui/src/views/connectors/types.ts

```diff
@@ -142,3 +133,8 @@ export enum ConnectorSelectionType {
   NEW = 'new',
   EXISTING = 'existing'
 }
+
+export enum ConnectorEntityIntent {
```

Comment:
Isn't this something Secrets would also use? In general, any entity might be created or edited. If yes, then lets name it something less specific to connectors.

Reply:
it would be changed to EntityIntent